src/Module/BaseApi.php

```diff
@@ -177,14 +179,21 @@ protected static function getCurrentUserID()
 		return (int)self::$current_user_id;
 	}
 
-	private static function checkBearer(string $authorization)
+	/**
+	 * Get the user id via the Bearer token
+	 *
+	 * @return int User-ID
+	 */
+	private static function getUserByBearer()
 	{
-		return (substr($authorization, 0, 7) == 'Bearer ');
-	}
+		$authorization = $_SERVER['HTTP_AUTHORIZATION'] ?? '';
+		$authorization = $_SERVER['AUTHORIZATION'] ?? $authorization;
```

Comment:
The `$authorization` variable isn't passed to `getUserByBearer()` anymore so it can't be assigned to itself.

Reply:
Ah, never mind, I didn't see that `$authorization` was declared in the previous line.